fbw-a380x/src/wasm/systems/a380_systems_wasm/src/payload.rs

```diff
@@ -79,6 +79,18 @@ pub(super) fn payload(builder: &mut MsfsAspectBuilder) -> Result<(), Box<dyn Err
         Variable::aircraft("PAYLOAD STATION WEIGHT", "Pounds", 18),
         Variable::aspect("PAYLOAD_STATION_18_REQ"),
     );
+    builder.copy(
+        Variable::aircraft("INTERACTIVE POINT OPEN", "percent", 0),
+        Variable::aspect("INTERACTIVE_POINT_0_OPEN_REQ"),
+    );
+    builder.copy(
+        Variable::aircraft("INTERACTIVE POINT OPEN", "percent", 2),
+        Variable::aspect("INTERACTIVE_POINT_2_OPEN_REQ"),
+    );
+    builder.copy(
+        Variable::aircraft("INTERACTIVE POINT OPEN", "percent", 10),
+        Variable::aspect("INTERACTIVE_POINT_10_OPEN_REQ"),
+    );
```

Comment:
You actually don't need this. You can add calls to `provides_aircraft_variable` (see lib.rs:290).